src/app/questionGenerator.js

```diff
@@ -0,0 +1,44 @@
+class QuestionGenerator {
+
+    //argument argAmountOfId will be relaced by connection to CurrentMode and it's IDs'
+    static generateQuestionsForCurrentMode(amountOfId) {
```

Comment:
What if the amountOfId comes in as null/undefined or string? What could you do to handle these situations? Could we exit the function earlier if the amoutOfId comes in in a wrong type?

Reply:
done! will be in new Draft Pull Request